.github/workflows/golangci.yml

```diff
@@ -11,10 +11,10 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.16.x
+          go-version: 1.17.x
 
       - name: Update dependencies
-        run: go mod tidy
+        run: go mod tidy -go=1.17
```

Comment:
We could keep it backward compatible and keep testing against `[1.15.x, 1.16.x, 1.17.x]` (2 "old" versions are probably enough) + bump the go version in go.mod. No?

Reply:
That's exactly the advice I was looking for, thx :blush: I did the same for private repos, but I wasn't sure whether 1.17 only was ok for this public one. I'll do.

> bump the go version in go.mod

You mean, using the earliest go version among the test ones (in this case, Go 1.15)?

Reply:
Yes, the version in go.mod (and the ones more recent) should be tested IMO